### PR TITLE
MODULES-2503 re-writing license

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: spec/acceptance/netapp_license_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ensure => present,
+    codes  => "saiufbaisubfiuasbfiuabs",
+  }
+}
+node 'vserver-01' {
+}
+    EOS
+    make_site_pp(pp)
+    run_device(:allow_changes => true)
+    run_device(:allow_changes => false)
```
Comment (hunner) [inline spec/acceptance/netapp_license_spec.rb]: I assume this wouldn't actually work? Does the API raise failures, or just continue to change?

Comment (tphoney) [inline spec/acceptance/netapp_license_spec.rb]: @hunner 
agreed, i am not sure we can write a test for this and have it public. i will contact the customer to see if they can test it.

should i comment this out. or make it optional as part of a test run ?

Comment (hunner) [inline spec/acceptance/netapp_license_spec.rb]: I'd probably expect_failure and maybe even match on the expected failure stdout, or just say "oh well" and comment it out. Either way :)

**Comment by tphoney** [comment]
thanks!

**Comment by tphoney** [comment]
@hunner commented out the test and gave an explanation, moved the test to make sure it wont auto run.

**Pull request merged**